test-imagemagick.js

```diff
@@ -0,0 +1,193 @@
+var uploadfs = require('./uploadfs.js')();
```

Comment:
Not sure why this is new here... was this inadvertently removed previously? I agree it is not a priority to mocha-ize it etc., please don't do that right now.